pkg/cluster/backup_manager.go

```diff
@@ -77,10 +77,10 @@ func (bm *backupManager) setupStorage() (s backupstorage.Storage, err error) {
 	b := cl.Spec.Backup
 	switch b.StorageType {
 	case spec.BackupStorageTypePersistentVolume, spec.BackupStorageTypeDefault:
-		if c.PVProvisioner == constants.PVProvisionerNone {
+		if c.PVProvisioner == constants.PVProvisionerNone && b.StorageClass == "" {
 			return nil, errNoPVForBackup
 		}
-		s, err = backupstorage.NewPVStorage(c.KubeCli, cl.Name, cl.Namespace, c.PVProvisioner, *b)
+		s, err = backupstorage.NewPVStorage(c.KubeCli, cl.Name, cl.Namespace, c.PVProvisioner, b.StorageClass, *b)
```

Comment:
probably convert PVProvisioner to storage class here before passing into the NewPVStorage func.

NewPVStorage can only take the storage class as a parameter.